connector/server/survivalanalysis/survival_query.go

```diff
@@ -223,10 +236,17 @@ func (q *Query) Validate() error {
 	if q.StartConcept == "" {
 		return fmt.Errorf("emtpy start concept path")
 	}
-	q.StartModifier = strings.TrimSpace(q.StartModifier)
-	if q.StartModifier == "" {
-		logrus.Warn("empty start concept, defaulte to \"@\"")
+	if q.StartModifier != nil {
+		*q.StartModifier.ModifierKey = strings.TrimSpace(*q.StartModifier.ModifierKey)
+		if *q.StartModifier.ModifierKey == "" {
+			logrus.Errorf("empty start modifier key")
```

Comment:
Those logged error should some context (e.g. query ID here). 

Also why do you log error but don't return an error and continue the execution?